NakedJavaScript/mensa-projekt
# Add working table sorting / searching and pager

# What has changed?

* Added Jquery Tablesorter Plugin
* The Userlist now shows you with a little highlight who is admin and who's not / also added thead that shows if user is admin or not
* You can sort by each thead / they are also sorted auto in the beginning
* You can search in each thead
* In the Food and Userlist there is also now a new sitepager for the list, removed the old one as the table sorter plugin has a cleaner solution

# How to test?

* Go to Userlist, you should now see the new table design where you can sort / search and click through pages
* The table should be sorted by the first entry (in this case the forename)
* Sort the table
* Search the table
* Click through the pages etc
* Go to foodlist, do the same 
* Go to your profile, then orders: do the same (note: I did not implement the table pager in the profile, I doubt we need it in there)
